soccer/gameplay/skills/angle_receive.py

```diff
@@ -16,24 +16,12 @@
 # Kick is a single_robot_behavior, so no need to import both
 class AngleReceive(skills.pass_receive.PassReceive):
```

Comment:
Angle receive captures the ball at a heading offset. This is used in one touch passes as the receiver. With the current changes, I don't think this would work in it's current state. It's ok to push this off as an issue, but we should just be specific that this is the decision being made so the AI can be updated to remove one touches.